packages/wonder-blocks-banner/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "exclude": ["dist"],
+    "extends": "../tsconfig-shared.json",
+    "compilerOptions": {
+        "outDir": "./dist",
+        "rootDir": "src",
+    },
+    "references": [
+        {"path": "../wonder-blocks-button"},
```

Comment:
Are these generated automatically?

Reply:
I wrote them all by hand.  I copy/pasted them and then tweaked the references.  That works in the short term, but in the long term we need tooling to make sure that these are correct.  It should be pretty obvious if there's a missing reference path though since the types for that dependency won't be available.